Velocity_Model/scripts/get_z.py

```diff
@@ -22,9 +22,16 @@
         "-v",
         "--version",
         help="VM version to use",
-        default="2.06",
+        default="2.07",
         choices=vm_versions.keys(),
     )
+    parser.add_argument(
+        "--no_header",
+        dest="with_header",
```

Comment:
Perhaps calling this keep_header (or something else), with header doesn't sound quite right.

Reply:
sure -